docs/ops/arithmetic/Round_5.md

```diff
@@ -0,0 +1,46 @@
+## Round <a name="Round"></a> {#openvino_docs_ops_arithmetic_Round_5}
+
+**Versioned name**: *Round-5*
+
+**Category**: Arithmetic unary operation 
+
+**Short description**: *Round* performs element-wise round operation with given tensor.
+
+**Detailed description**: Operation takes one input tensor and rounds the values, element-wise, meaning it finds the nearest integer for each value. In case of halfs, the rule is to round them to the nearest even integer.
+    
+    round([-4.5, -1.9, -1.5, 0.9, 1.5, 2.3, 2.5, 1.5]) = [-4.0, -2.0, -2.0, 1.0, 2.0, 2.0, 2.0, 2.0]
+
+**Attributes**:
+
+    No attributes available.
```

Comment:
Why don't we propose another mode - *round half away from zero*? Here you can propose only round half to even.
For Pytorch I see that they implement *round half away from zero*.

Reply:
Maybe it makes sense in the description to refer to IEEE754 (that describes *round half to even*) and IEEE754-2008 (*round half away from zero*). And please investigate which mode should be default.

Reply:
From CPU plugin perspectives, it may be difficult to implement *round half to even* case. Let us support to support both modes and generate IR with original mode and plugin will change it if needed.

Reply:
I add attribute `mode` with values `half_to_even` (default) and `half_away_from_zero`. Examples also updated .